qa/Gemfile

```diff
@@ -1,5 +1,5 @@
 source "https://rubygems.org"
 gem "runner-tool", :git => "https://github.com/purbon/runner-tool.git"
 gem "rspec", "~> 3.1.0"
-gem "rake"
+gem "rake", "12.1.0"
```

Comment:
Should this be `~> 12.1.0` to allow us to discover patch releases (e.g., for security fixes), or is a hard-pin nexessary?

Reply:
I don't think rake even tries to follow semver. :) 

For the 5.x branch, I think we should be explicit as possible to help ensure nothing sneaks into a release (or causes instability in the release process).